stablehlo/tools/StablehloInterpreterMain.cpp

```diff
@@ -76,6 +81,30 @@ TranslateFromMLIRRegistration stablehlo_interpreter(
       registry.insert<stablehlo::StablehloDialect>();
     });
 
+llvm::cl::opt<std::string> _target(
+    "target", llvm::cl::desc("Target version for serialization"),
+    llvm::cl::init(""));
+
+TranslateFromMLIRRegistration stablehlo_serialize(
```

Comment:
How about "serializeRegistration"? Similarly, for "interpretRegistration" and "deserializeRegistration".